imagebuildah/util_test.go

```diff
@@ -0,0 +1,86 @@
+package imagebuildah
+
+import (
+	"archive/tar"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/opencontainers/go-digest"
+)
+
+func TestGeneratePathChecksum(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "testdir")
```

Comment:
Use a value returned by `t.TempDir()`.  It was introduced in Go 1.15, and you don't need to clean it up.